Include/py_curses.h

```diff
@@ -64,7 +64,7 @@ typedef struct {
     char *encoding;
 } PyCursesWindowObject;
 
-#define PyCursesWindow_Check(v)  (Py_TYPE(v) == &PyCursesWindow_Type)
+#define PyCursesWindow_Check(v) Py_IS_TYPE((v), &PyCursesWindow_Type)
```

Comment:
`(v)` parenthesis looks useless:

```suggestion
#define PyCursesWindow_Check(v) Py_IS_TYPE(v, &PyCursesWindow_Type)
```